internal/service/lexv2models/slot_test.go

```diff
@@ -143,6 +143,41 @@ func TestAccLexV2ModelsSlot_ObfuscationSetting(t *testing.T) {
 	})
 }
 
+func TestAccLexV2ModelsSlot_SubSlotSetting(t *testing.T) {
```

Comment:
```suggestion
func TestAccLexV2ModelsSlot_subSlotSetting(t *testing.T) {
```

Tiny nit. Go naming convention: `Xyz_Abc_jkl` or `Xyz_jkl`. If you have 3 parts, "Xyz" is book title, "Abc" is chapter title, "jkl" is section title. With just 2, book, section. The last one, whether you have 4, 3, or 2, should be lowercase.